docs/components/checkbox/index.md

```diff
@@ -0,0 +1,55 @@
+# Checkbox
+
+Provides a Toucan-styled [checkbox element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox). If you are building forms, you may be interested in the CheckboxField component instead.
+
+## Checked Value
+
+To set the `checked` attribute of the checkbox, provide `@value`.
+
+```hbs
+<Form::Controls::Checkbox @value={{true}} />
+<Form::Controls::Checkbox @value={{false}} />
+```
+
+## Indeterminate
+
+Checkboxes have the ability to be in the [indeterminate state](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#indeterminate_state_checkboxes). This is accomplished with `@isIndeterminate`. It is up to the consumer to decide what will force the checkbox into the indeterminate state. The [data-checked](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-checked) attribute is set to either "true", "false", or "mixed" depending on the state (where "mixed" is set if `@isIndeterminate={{true}}`).
+
+```hbs
+<Form::Controls::Checkbox @isIndeterminate={{true}} />
+```
+
+## onChange
+
+To tie into the input event, provide `@onChange`. `@onChange` will return three arguments:
+
+1. the checked attribute from the target
```

Comment:
Would this be the checked state before or after the change? Might be worth mentioning that here

Reply:
Sure thing - thank you!